specs/using_grafana_for_monitoring_dashboards.adoc

```diff
@@ -0,0 +1,132 @@
+// vim: tw=79
+
+= Spec - Using Grafana for displaying monitoring dashboard
+
+Grafana provides a rich set of tools to display monitoring and analytics using
+time series data.
+
+== Problem description
+
+Tendrl currently uses set of API's which are converted to a consumeable form
+for the front-end to display monitoring data. The front-end is unable to handle
+large amounts of data as the interval for the graphs increase and needs
+continuous polling to update the graphs in real-time. The graphs don't provide
+visual updates about a on going alert state.
+
+Tendrl also does not provide a way to customize the views for the end users
+needs, they are expected to use the default views provided.
+
+== Use Cases
+
+* End users are provided with more detailed graphs with possibility of querying
+based on any time interval.
+
+* Users can create their own graphs which suit their use cases.
+
+* The graphs provides visual real time updates about an ongoing alert in the
+system.
+
+== Proposed change
+
+Use Grafana to display monitoring, analytics and alerting dashboards.
+
+=== Alternatives
+None
+
+=== Data model impact:
+None
```

Comment:
@brainfunked the data model changes are in a different spec. https://github.com/Tendrl/specifications/pull/206, I have added the link in the spec.